### ISSUE TITLE
Client cannot login. no errors, page just refreshes

### ISSUE THREAD

**Issue by chrisg39** [osTicket/osTicket-1.7#729]
hey guys,
I am trying to incorporate osTicket (v1.7.1.2) into wordpress 3.6.0 using the method outlined here http://www.kodelibrary.com/tag/osticket-wordpress-guide/
I thought everything worked ok... I was able to get the css cleaned up and the admin and staff panels work fine.
I ran into an issue when i tried to read an opened ticket from the client side. if the ticket number and email are not entered correctly the form spits out an error message and does not allow the user to access the ticket just as expected. when a valid email and ticket number input the page just refreshes with no error message and no ticket is displayed. I have found similar issues but none of them seem to apply to what i am experiencing. any help would be greatly appreciated.

**Comment by protich** [comment]
Sorry but this is not something we can help with - get the developer/writer of the guide to make it up to date.

**Comment by chrisg39** [comment]
ok thanks a bunch i thought it might be a no-brainer for you guys.

**Comment by ntozier** [comment]
There may be a error being logged in your php error log and its just not
getting displayed. You could also try turning on error display in osTicket.

What Peter is trying to say is that we do not support the mod that you are
using and should really talk to the mod author for support.  Additionally
its not really a osTicket bug as osTicket works with out using the mod and
since we dont use Wordpress or the mod we cannot really support it.  (Does
that make sense?)

**Comment by chrisg39** [comment]
absolutely, i know it is a little off topic and with that being said i especially appreciate the replies... Unfortunately i cant find any errors in the osticket log just the regular User Login entry [...ail.com/903610 logged in [19...]
I will see what else i can find
